src/app/components/Label/Label.styles.tsx

```diff
@@ -22,11 +22,15 @@ const getColor = (color?: ColorType, variant?: string) => {
 
 	const colors = {
 		danger: () => tw`text-theme-danger-400 border-theme-danger-100 dark:border-theme-danger-400`,
+		"danger-bg": () =>
+			tw`text-[#A56D4C] bg-[#FBEED6] dark:border dark:text-[#F39B9B] dark:bg-transparent dark:border-[#AA6868]`,
```

Comment:
same question here, not a fan of hex codes for colors 🤔 if they are different in the design, is there any color in the palette close enough to use for it instead?